docs/roadmap/2023-11-dbt-tng.md

```diff
@@ -0,0 +1,107 @@
+| [v1.6](https://docs.getdbt.com/guides/migration/versions/upgrading-to-v1.6) | July | [Quiara Alegría Hudes](https://github.com/dbt-labs/dbt-core/releases/tag/v1.6.0#:~:text=Quiara%20Alegr%C3%ADa%20Hudes%20(b.%201977)) | New Semantic Layer spec. More on model governance (deprecations). Saving time and $$ with retry + clone. Initial rollout of materialized views. |
+| [v1.7](https://docs.getdbt.com/guides/migration/versions/upgrading-to-v1.7) | November | <a href="https://github.com/dbt-labs/dbt-core/releases/tag/v1.7.0#:~:text=%238692)-,Questlove%20(b.%201971),-Thanks%20to%20%40dave">Questlove</a> | More flexible access to "applied state" in docs generate and source freshness. Improvements to model governance & semantic layer features (driven by user feedback). |
+
+We added a **lot** of stuff this year! Over the past three dbt Core minor versions, we’ve managed to knock out a litany of the most popular issues and discussions gathered over the past several years:
+
+- [CLI preview](https://github.com/dbt-labs/dbt-core/discussions/5418) (1.5)
+- [Invoking dbt as a Python module](https://github.com/dbt-labs/dbt-core/issues/2013) (1.5)
+- [Materialized views](https://github.com/dbt-labs/dbt-core/issues/1162) (1.6)
+- [Namespacing for dbt resources](https://github.com/dbt-labs/dbt-core/issues/1269) (1.6), in support of [multi-project collaboration](https://github.com/dbt-labs/dbt-core/discussions/6725)
+- [`docs generate --select` for slimmer catalog queries](https://github.com/dbt-labs/dbt-core/issues/6014) (1.7)
+- And… we’re finally taking aim at [unit testing for dbt-SQL models](https://github.com/dbt-labs/dbt-core/discussions/8275) (!), coming in 1.8, which you should read more about in the section below.
+
+Thank you for providing your upvotes, comments, and feedback. One of the best things about building dbt Core in the open is that we are all pushing forward the analytics engineering standard together. We’re able to prioritize these features and paper cuts because of your participation. 
+
+We’ve got lots more to build - there are some highly upvoted issues and discussions that remain, and gaps in the analytics engineering workflow that we want to close. But before we keep building, we must ensure our foundation is a **stable** one.
+
+# The next six months: stability (& unit testing!)
+
+| Version | When | Stuff | Confidence |
+| --- | --- | --- | --- |
+| 1.8 | Spring 2024 | Stable interfaces for adapters & artifacts. Built-in support for unit testing dbt models. | 80% |
+
+Since the v1.0 release of dbt-core (almost two years ago), we’ve released a minor version of `dbt-core` every three months. The January release (post-Coalesce, post-holidays) tends to be an understated affair: tech debt, bug fixes, support for Python 3-dot-new.
```

Comment:
```suggestion
Since the v1.0 release of dbt Core (almost two years ago), we’ve released a minor version of dbt Core every three months. The January release (post-Coalesce, post-holidays) tends to be an understated affair: tech debt, bug fixes, support for Python 3-dot-new.
```